### DISCUSSION TITLE
Drop the Debug messages

### DISCUSSION THREAD

**Discussion by advapiIT** [datalust/seq-tickets#1797] [Help]
Hello,
I've made the error to configure 2 apps to save also the debug log to seq... this caused the storage on azure to finish... I was wondering since we don't have any available slot to deploy the applications with serilog configured to just send Information and higher, if it's possible somehow to force seq to delete all the debug and trace messages?

Or at least to stop ingesting them?

Thanks

**Comment by nblumhardt** [comment]
Hi! Adding a filter to your [API key(s)](https://docs.datalust.co/docs/api-keys), via Data > Ingestion > (input receiving debug events) should do it:

```
@Level not in ['Debug', 'Trace']
```

To remove the existing problematic events, setting up a [retention policy](https://docs.datalust.co/docs/retention-policies) is the best option.

Hope this helps,
Nick
